notebooks/a01_manifold_class.ipynb

```diff
@@ -0,0 +1,2260 @@
+{
```

Comment:
<p>We could introduce this by saying: New manifolds can also be created by composing existing manifolds M1, M2, ... .</p><p> </p><p>Technically, this would be the definition of the Nfold manifold. <code>NFold</code> class in geomstats.</p><p> </p><p>A product manifold can be the product of different manifolds: M1, M2, etc</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1340/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>done</p>